src/main/resources/spring/camel-context.xml

```diff
@@ -169,13 +199,14 @@
                     <to uri="direct:check-rbac-permissions" />
                     <!-- RBAC check end-->
 
+                    <to uri="direct:to-pageBean"/>
                     <choice>
                         <when>
                             <simple>${header.filterText} == null</simple>
-                            <bean ref="analysisService" method="findAllByOwner(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.page}, ${header.size})" />
+                            <bean ref="analysisService" method="findAllByOwner(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.pageBean})" />
```

Comment:
I think it will be required to add `<bean ref="apiService" method="mapToCustomPage(${exchange})"/>` to transform the response to the new Page Format.

Reply:
+1